### DISCUSSION TITLE
All Records not visible which loading the poly lookup

### DISCUSSION THREAD

**Discussion by codewithjigisha** [khoait/DCE.PCF#215] [Q&A]
### Problem description

I have more than 100 records in the list from which user can select multiple records in poly lookup, but user only sees the first 50 records in the list which lead to confusion for them as they don't see it in the selection list. Also, I am using fetchXml not the system view.

### Feature description

Please suggest on how do it load more records in single go.

**Comment by khoait** [comment]
Hi @codewithjigisha , thanks for your question.

the component will apply the page size configuration in user personal settings. User can set this value to max 250 per page. Please see more here: https://github.com/khoait/DCE.PCF/wiki/PolyLookup#suggestions

And pagination here: https://github.com/khoait/DCE.PCF/wiki/PolyLookup#pagination

The intend is to leverage the searching feature which can cover all use cases, doesn't matter how big is the table. there are indications at the end of the list to show if there are more results or not.

This is for the component to perform better and provide a better user experience as users need to refine the list using the search box instead of browsing through multiple pages in the suggestion list.

Hope it helps!
